ActionStorage-master/src/test/java/app/just/actionstorage/repository/UserRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,20 @@
+package app.just.actionstorage.repository;
+
+import app.just.actionstorage.entity.UserEntity;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
+public class UserRepositoryIntegrationTest extends AbstractTest {
+  @Autowired
+  private UserRepository userRepository;
+
+  @Test
```

Comment:
I don`t see data comparing. You need give data for database. For this use DBRIDER